lib/y2lan_restart_common.pm

```diff
@@ -38,6 +43,21 @@ our @EXPORT = qw(
 );
 my $module_name;
 
+=head2 initialize_y2lan
+
+ initialize_y2lan();
+
+Initialize yast2 lan.
```

Comment:
better in one line all the comments or if you find some more fancy way in Perl, up to you.

Reply:
script/generateLibDocs.sh has problem to handle this shorten sentences without empty line between lines. Anyway this should be clear enough to read.